src/WebJobs.Script/Workers/SharedMemoryDataTransfer/NullSharedMemoryManager.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers.SharedMemoryDataTransfer
+{
+    public class NullSharedMemoryManager : ISharedMemoryManager
```

Comment:
CI build is failing.
` 'NullSharedMemoryManager' does not implement interface member 'ISharedMemoryManager.TryTrackSharedMemoryMap(string)'`

Please implement that and try a local build to make sure.

Reply:
We should also check if things work fine in cases where we don't create a `FunctionDataCache` service - running the end to end tests might show that. 
Otherwise, we could implement a `NullFunctionDataCache` and make sure to return `IsEnabled` as `False`.